services/__init__.py

```diff
@@ -0,0 +1 @@
+from services import *
```

Comment:
I'm getting the impression this is boilerplate, as this one-liner (AFAICT) doesn't do anything.

For more detail, have a [read about `from module import *`](https://docs.python.org/3/tutorial/modules.html#importing-from-a-package).

Reply:
This is extremelyyy, tricky! I will try to get used to that. Thought I had to have them defined in this file to compile. =O

Thanks Rob!